app/views/state_file/questions/income_review/edit.html.erb

```diff
@@ -27,6 +27,22 @@
     </div>
   <% end %>
 
+  <% if current_intake.state_file1099_rs.present? %>
```

Comment:
this will probably get caught in acceptance testing, but the 1099R goes before the 1099G (and will be after W2)